demo/image-carousel-demo-app.html

```diff
@@ -6,7 +6,7 @@
     <style>
     </style>
     <div>
-      <image-carousel id="my-carousel" aria-label="actual image-carousel">
+      <image-carousel id="my-carousel" notify-selection-changed="true" aria-label="actual image-carousel">
```

Comment:
No need to add `true` only writing attribute `notify-selection-changed` should do the trick.